apps/server/src/modules/account/services/account.service.spec.ts

```diff
@@ -235,7 +246,14 @@ describe('AccountService', () => {
 	describe('validatePassword', () => {
 		const setup = () => {
 			configService.get.mockReturnValue(true);
-			return new AccountService(accountServiceDb, accountServiceIdm, configService, accountValidationService, logger);
+			return new AccountService(
```

Comment:
Is already written in this way, but switch to the way that we have on all other places will make sense or?
let accountService: AccountService;
accountService = module.get(AccountService);